src/components/form/base-controls/checkbox-control/types.ts

```diff
@@ -12,7 +12,8 @@ type InheritedInputProps = Pick<
 >
 
 interface CheckboxControlProps extends InheritedInputProps {
-	id: NativeInputProps['id']
+	id?: NativeInputProps['id']
+	onChange?: NativeInputProps['onChange']
```

Comment:
Yes, good call! And I don't remember exactly 🤔 but I believe it's also optional on the native type?

Reply:
Yes the `id` prop is optional on the native type! But I think we should keep it required so that when the `CheckboxControl` is used outside of it's associated `Field` component, there's a requirement to associate it with a `<label>`.

Reply:
Makes sense, updated 😃

Reply:
Updated! 😃